lib/core/config.js

```diff
@@ -98,8 +104,8 @@ Config.prototype._updateBlockchainCors = function(){
   let webServerConfig = this.webServerConfig;
   let corsParts = [];
 
-  if(webServerConfig && webServerConfig.enabled) {
-    if(webServerConfig.host) corsParts.push(utils.buildUrlFromConfig(webServerConfig));
+  if(webServerConfig && webServerConfig.host) {
```

Comment:
Are we ignoring `webServerConfig.enabled` from now on?

Reply:
That's because if webserver is disabled, we don't setup the cors, but we need the cors still for web3 commands